sklearn/utils/testing.py

```diff
@@ -58,7 +58,6 @@
 
 from sklearn.base import (ClassifierMixin, RegressorMixin, TransformerMixin,
                           ClusterMixin)
-from sklearn.cluster import DBSCAN
```

Comment:
For mysterious reasons this import was causing trouble on my local version. Since it appears to no longer be used I removed it.